artview/plugins/background.py

```diff
@@ -328,4 +329,46 @@ def apply(self):
         self.VpyartDisplay.update(strong=False)
 
 
-_plugins = [TopographyBackground, ImageBackground]
+def topography_mode():
+    from ..modes import change_mode
+    change_mode(
+        [FileNavigator, RadarDisplay, TopographyBackground],
+        [
+            ((0, 'Vradar'), (1, 'Vradar')),
+            ((1, 'VpyartDisplay'), (2, 'VpyartDisplay')),
+            ]
+        )
+
+def background_mode():
+    from ..modes import change_mode
+    change_mode(
+        [FileNavigator, RadarDisplay, ImageBackground],
+        [
+            ((0, 'Vradar'), (1, 'Vradar')),
+            ((1, 'VpyartDisplay'), (2, 'VpyartDisplay')),
+            ((1, 'Vradar'), (2, 'Vradar')),
+            ((1, 'VplotAxes'), (2, 'VplotAxes')),
+            ]
+        )
+
+
+try:
+    from mpl_toolkits.basemap import shiftgrid, cm
```

Comment:
Eventually I'd like to replace basemap with cartopy or other to get ahead of the basemap deprecation in a couple of years.